packages/tabs/partials/tab-trigger/index.tsx

```diff
@@ -50,7 +52,7 @@ function TabTrigger({
       <span className={s.inner}>
         <span className="g-type-body-strong">{tab.heading}</span>
         {tab.tooltip && (
-          <Tooltip label={tab.tooltip} aria-label={tab.tooltip}>
+          <Tooltip label={tab.tooltip} aria-label={tab.tooltip} theme={theme}>
```

Comment:
Due to some quirks around `Portal` rendering, have to pass down `theme` prop for `<Tooltip />` (rather than being able to cascade in CSS custom properties as is done for all other `partials`).